typescript/infra/config/environments/mainnet3/core.ts

```diff
@@ -52,17 +74,30 @@ export const core: ChainMap<CoreConfig> = objMap(
       ...owner,
     };
 
+    // No static aggregation ISM support on zkSync
+    const defaultZkSyncIsm: RoutingIsmConfig = {
```

Comment:
nit: do we need to build this when isZkSyncChain is false? not a big deal if it's clunky to only do it conditionally